### PR TITLE
Filter context

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added playlist management, like, watch later and history

### CODE REVIEW

File: src/pages/PlaylistVideos/PlaylistVideos.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Navbar from "../../components/Navbar/Navbar";
```
Comment (saadahmadhere) [inline src/pages/PlaylistVideos/PlaylistVideos.jsx:2]: You can try barrel imports if you need to import many files. That would decrease the number of lines of code.

File: src/pages/Signup/Signup.jsx [hunk 2]
```diff
@@ -22,41 +22,41 @@ const Signup = () => {
...
-            <div class="form-buttons">
-              <button class="btn btn-primary signup-btn">Sign Up</button>
+            <div className="form-buttons">
+              <button className="btn btn-primary signup-btn">Sign Up</button>
             </div>
-            <div class="login-redirect-container">
-              <Link to="/login" class="login-redirect-link btn-link">
+            <div className="login-redirect-container">
+              <Link to="/login" className="login-redirect-link btn-link">
                 Already have an account{" "}
```
Comment (saadahmadhere) [inline src/pages/Signup/Signup.jsx]: remove {" "}

**Review by saadahmadhere — COMMENTED** [review]
Well written code. Nice use of useContext() to avoid prop drilling.

**Review by GuptaEesh — COMMENTED** [review]
Code looks good.
Good use of data modelling
Explanatory variable names.
Good to merge.

File: src/pages/PlaylistVideos/PlaylistVideos.jsx (hunk 1 above)
Comment (omgaikwad) [reply src/pages/PlaylistVideos/PlaylistVideos.jsx:2]: Sure!

**Pull request merged**